src/unittest/python/pmf_tests.py

```diff
@@ -55,24 +55,40 @@ def test_fit_assigns_by_default(self):
     def test_fit_does_not_assign_if_requested(self):
         ps = pmf.PsychometricFunction(pmf.logistic, 0.5)
         self.assertIsNone(ps.params)
-        ps.fit(self.data, assign=False)
+        ps.ml_fit(self.data, assign=False)
         self.assertIsNone(ps.params)
         self.assertIsNone(ps.data)
 
     @mock.patch('esdt.pmf.optimize.fmin')
     def test_fit_uses_starting_values_if_specified(self, mock_fmin):
         mock_fmin.return_value = np.ones(3, 'd')
         ps = pmf.PsychometricFunction(pmf.logistic, 0.5)
-        ps.fit(self.data, start='ANY_STARTING_VALUE')
+        ps.ml_fit(self.data, start='ANY_STARTING_VALUE')
         self.assertEqual(mock_fmin.mock_calls[0][1][1], 'ANY_STARTING_VALUE')
 
+    def test_fit_is_deprecated(self):
+        ps = pmf.PsychometricFunction(pmf.logistic, 0.5)
+        ps.ml_fit = mock.Mock()
+        with self.assertWarns(DeprecationWarning):
+            ps.fit('ANY_DATA', start='ANY_START')
+        ps.ml_fit.assert_called_once_with('ANY_DATA', start='ANY_START')
+
     def test_jackknife_sem_drops_every_block(self):
         ps = pmf.PsychometricFunction(pmf.logistic, 0.5)
-        ps.fit = mock.Mock()
-        ps.fit.side_effect = [(k + np.zeros(3, 'd'), 1) for k in range(5)]
+        ps.ml_fit = mock.Mock()
+        ps.ml_fit.side_effect = [(k + np.zeros(3, 'd'), 1) for k in range(5)]
         ps.params = np.zeros(3, 'd')  # Needed for influence
         ps.jackknife_sem(self.data)
-        self.assertEqual(len(ps.fit.mock_calls), self.data.shape[0])
+        self.assertEqual(len(ps.ml_fit.mock_calls), self.data.shape[0])
+
+    def test_deviance_test(self):
```

Comment:
I'm not sure what "test_deviance_test" means. Is it testing the deviance test to give correct answers?